### PR TITLE
Added support for hidden by mp in script arguments

### PR DESCRIPTION
**Pull request by omerKarkKatz**
## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-11782

## Description
Added the option to give a specific value for xsiam hidden attribute in script arguments.

### CODE REVIEW

File: .changelog/4550.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Added the ability to have a special value for the hidden attribute of a script argument.
```
Comment (samuelFain) [inline .changelog/4550.yml:2]: I feel this better corresponds with the agreed [changlog templates](https://confluence-dc.paloaltonetworks.com/display/DemistoContent/Changelogs). 
```suggestion
- description: Added support for a special value in the hidden attribute of a script argument.
```

**Review by samuelFain — APPROVED** [review]
LGTM.
Please see my comment about the changlog before merging.

File: .changelog/4550.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Added support for a special value in the hidden attribute of a script argument.
```
Comment (omerKarkKatz) [inline .changelog/4550.yml:2]:
```suggestion
- description: Added support for a unique value in the hidden attribute of a script argument.
```

File: .changelog/4550.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Added support for a unique value in the hidden attribute of a script argument.
```
Comment (omerKarkKatz) [inline .changelog/4550.yml:2]:
```suggestion
- description: Added support for a specific value in the hidden attribute of a script argument.
```

**Pull request merged**